worker/deploy/README.md

```diff
@@ -1,47 +1,123 @@
+
+![Worker Deployment](../../imgs/wd.png)
+
+## **Minimum Requirements**
+
+### _Worker Node_
+  The requirements of the Worker Node depend a lot on the workload of the Jobs you intend to submit to Arrebol. Below is a minimum standard recommendation:
+
+  - vCPU: 2
+  - RAM: 2GB
+  - FREE DISK SPACE: 10GB
+  - OS: Ubuntu 16 or higher
+
+### _Coordination Host_
+  - vCPU: 1
+  - RAM: 1GB
+  - FREE DISK SPACE: 10GB
+  - OS: Ubuntu 16 or higher
+
+## **Setup**
+
+### _1.Create RSA Key Pair_
+
+`Note: If you already have a key pair configured between the coordination host and the worker nodes go to Install dependencies.`
+
+Log into the coordination host, and then use command-line SSH to generate a key pair using the RSA algorithm.
+To generate RSA keys, on the command line, enter:
+
+```bash
+mkdir -p ~/.ssh
+cd ~/.ssh
+ssh-keygen -t rsa -N "" -f coordination_host_key
+chmod 600 coordination_host_key
+```
+The coordination host private key will be generated using the filename 
+**coordination_host_key** and the public key using the filename **coordination_host_key.pub**. It was stored in a **.ssh** directory off home directory.
+
+Now you need to authorize the coordination host to access the worker node via SSH.
+
+First you need have the **coordination_host_key.pub** in the worker node. After, log into the worker node machine and write the **coordination_host_key.pub** into **authorized keys** using the commands below:
 ```bash
+mkdir -p ~/.ssh
+touch ~/.ssh/authorized_keys
+cat ~/coordination_host_key.pub >> ~/.ssh/authorized_keys
+```
+
+
+
+### _2.Install dependencies_
+
+Log in the coordination host and run the following commands to install dependencies.
+
+```
 sudo apt update
 sudo apt install software-properties-common
 sudo apt-add-repository --yes --update ppa:ansible/ansible
-sudo apt install -y ansible
+sudo apt install -y ansible git grep sed 
 ```
 
-`Note: The deployment requires versions newer than 14 for ubuntu.`
+### _3.Download the repository_
+
+Log in the coordination host and run the below commands to download the arrebol repository.
+
+```bash
+git clone -b feature/remote-worker https://github.com/ufcg-lsd/arrebol.git
+```
 
-## Configuration
+### _4.Fill configuration file_
 
-The `hosts.conf` configuration file should be edited to declare the Worker Node. See below how to edit it.
+Go to the directory _worker/deploy_ inside _arrebol_ directory.
 
-### Example
+```bash
+cd arrebol/worker/deploy
+```
+
+Then, edit the `hosts.conf` file present in this directory, as instructed in the following.\
+ **All fields** of the files must be filled. See below how to edit it. 
+
+#### Hosts configuration
 ```
+# Required
+# For each worker node write a field started by `worker_ip`
+
 worker_ip_1=10.30.1.36
 worker_ip_2=10.30.1.37
 
-remote_user=ubuntu
-
-ansible_ssh_private_key_file=/home/admin/.ssh/priv_key
+# Required
```

Comment:
add a comment explaining what is a remote user, also add ubuntu as the default one (i guess this is the one we used to use right?)

Reply:
done.

Reply:
done.